### PR TITLE
Added documentation to properly support XDK in WebViews

### PR DESCRIPTION
**Pull request by imankur**
Added documentation to properly support XDK in WebViews

### CODE REVIEW

File: docs/web-sdk/support-native.md [hunk 1]
```diff
@@ -19,6 +19,40 @@ The following settings on the WebView are also necessary to ensure the SDK has t
...
+                using: { note in
+                    // App moved to background!
+                    self.webView?.evaluateJavaScript("window.HaptikSDK.pause();", completionHandler: nil)
+            })
+    }
+    
+    deinit {
+        // Remove Notification observer
+        NotificationCenter.default.removeObserver(self, name: UIApplication.willResignActiveNotification, object: nil)
+    }
```
Comment (tejas-haptik) [inline docs/web-sdk/support-native.md:57]: All of this code is in single file right?

Comment (imankur) [reply docs/web-sdk/support-native.md:57]: Yes.

File: docs/web-sdk/support-native.md [hunk 2]
```diff
@@ -19,6 +19,40 @@ The following settings on the WebView are also necessary to ensure the SDK has t
...
+    super.onStop()
+    webview.loadUrl("javascript:window.HaptikSDK.pause();")
+}
+```
+
 ## iOS
 
 iOS doesn't need any extra permissions in order to operate. Just open the html file where you've configured the SDK inside the app.
+
+Just like Android, in iOS as well the lifecycle of a WebPage is coupled to the iOS app lifecycle. And we need to notify Haptik SDK when the app goes in background. To do this add inactive state observer in your view controller which consists WkWebView and add following code :
```
Comment (tejas-haptik) [inline docs/web-sdk/support-native.md:35]: `And we need to...` --> `We need to...`

Comment (tejas-haptik) [reply docs/web-sdk/support-native.md:35]: `To do this add...` --> `To do this, please add...`

Comment (tejas-haptik) [reply docs/web-sdk/support-native.md:35]: `add following code :` --> `add the following code:`

File: docs/web-sdk/support-native.md [hunk 3]
```diff
@@ -19,6 +19,40 @@ The following settings on the WebView are also necessary to ensure the SDK has t
...
+    super.onStop()
+    webview.loadUrl("javascript:window.HaptikSDK.pause();")
+}
+```
+
 ## iOS
 
 iOS doesn't need any extra permissions in order to operate. Just open the html file where you've configured the SDK inside the app.
+
+Just like Android, in iOS as well the lifecycle of a WebPage is coupled to the iOS app lifecycle. We need to notify Haptik SDK when the app goes in background. To do this add inactive state observer in your view controller which consists WkWebView and add following code :
```
Comment (tejas-haptik) [inline docs/web-sdk/support-native.md:35]: `To do this add...` -> `To do this, please add...`

Comment (tejas-haptik) [reply docs/web-sdk/support-native.md:35]: `which consists WkWebView...` --> `which consists the WkWebView...`

Comment (tejas-haptik) [reply docs/web-sdk/support-native.md:35]: `add following code :` --> `add the following code:`

**Pull request merged**